### DISCUSSION TITLE
部署策略 - DaoCloud Enterprise

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1078] [Announcements]
# 部署策略 - DaoCloud Enterprise

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kairship/policy/propagation-policy/

**Comment by liyuerich** [comment]
部署策略中有个优先级，没有介绍

**Reply by liyuerich** [reply]
部署策略有“更新”操作，能否描述一下

**Comment by ModetaNiu** [comment]
Already fixed in PR #1113

**Comment by ModetaNiu** [comment]
Already fixed in PR #1113
